Dockerfile

```diff
@@ -1,15 +1,19 @@
 FROM ubuntu:23.04
 
-ENV LD_LIBRARY_PATH=/usr/lib64:$LD_LIBRARY_PATH
-ENV LIBRARY_PATH=/usr/lib64:$LIBRARY_PATH
+#use --build-arg LIB_DIR=/usr/lib for arm64 cpus
+ARG LIB_DIR=/usr/lib64
+ENV NODE_ENV=$BUILD_DEVELOPMENT
```

Comment:
thanks you send mr, Is this environment variable useful?

Reply:
nice catch! No, I have left it by mistake.

Reply:
could you merge the two commit into one? I think they should be a commit

Reply:
agreed, I was assuming you would do that when merging PR, but anyway I've done it.

Reply:
Thank you for your contribution

Reply:
Thank you for this awesome project!